src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminCourtContactEndpointTest.java

```diff
@@ -0,0 +1,243 @@
+        final var response = given()
+            .relaxedHTTPSValidation()
+            .header(CONTENT_TYPE, CONTENT_TYPE_VALUE)
+            .header(AUTHORIZATION, BEARER + forbiddenToken)
+            .body(getTestContactsJson())
+            .when()
+            .put(ADMIN_COURTS_ENDPOINT + BIRMINGHAM_CIVIL_AND_FAMILY_JUSTICE_CENTRE_SLUG + CONTACTS_PATH)
+            .thenReturn();
+
+        assertThat(response.statusCode()).isEqualTo(FORBIDDEN.value());
+    }
+
+    @Test
+    public void shouldRetrieveAllContactTypes() {
+        final var response = given()
+            .relaxedHTTPSValidation()
+            .header(CONTENT_TYPE, CONTENT_TYPE_VALUE)
+            .header(AUTHORIZATION, BEARER + authenticatedToken)
+            .when()
+            .get(ADMIN_COURTS_ENDPOINT + CONTACT_TYPES_PATH)
+            .thenReturn();
+
+        assertThat(response.statusCode()).isEqualTo(OK.value());
+        final List<ContactType> contactTypes = response.body().jsonPath().getList(".", ContactType.class);
+        assertThat(contactTypes).hasSizeGreaterThan(1);
+    }
+
+    @Test
+    public void shouldRequireATokenWhenRetrievingAllContactTypes() {
+        final var response = given()
+            .relaxedHTTPSValidation()
+            .header(CONTENT_TYPE, CONTENT_TYPE_VALUE)
+            .when()
+            .get(ADMIN_COURTS_ENDPOINT + CONTACT_TYPES_PATH)
+            .thenReturn();
+
+        assertThat(response.statusCode()).isEqualTo(UNAUTHORIZED.value());
+    }
+
+    @Test
+    public void shouldBeForbiddenForRetrievingContactTypes() {
+        final var response = given()
+            .relaxedHTTPSValidation()
+            .header(CONTENT_TYPE, CONTENT_TYPE_VALUE)
+            .header(AUTHORIZATION, BEARER + forbiddenToken)
+            .when()
+            .get(ADMIN_COURTS_ENDPOINT + CONTACT_TYPES_PATH)
+            .thenReturn();
+
+        assertThat(response.statusCode()).isEqualTo(FORBIDDEN.value());
+    }
+
+    private List<Contact> getCurrentContacts(final String slug) {
+        final var response = given()
+            .relaxedHTTPSValidation()
+            .header(CONTENT_TYPE, CONTENT_TYPE_VALUE)
+            .header(AUTHORIZATION, BEARER + authenticatedToken)
+            .when()
+            .get(ADMIN_COURTS_ENDPOINT + slug + CONTACTS_PATH)
+            .thenReturn();
+
+        return response.body().jsonPath().getList(".", Contact.class);
+    }
+
+    private List<Contact> addNewContact(List<Contact> contacts) {
+        List<Contact> updatedContacts = new ArrayList<>(contacts);
+        updatedContacts.add(TEST_CONTACT);
+        return updatedContacts;
+    }
+
+    private List<Contact> removeContact(List<Contact> contacts) {
+        List<Contact> updatedContacts = new ArrayList<>(contacts);
+        updatedContacts.removeIf(time -> time.getNumber().equals(TEST_NUMBER));
+        return updatedContacts;
+    }
+
+    private static String getTestContactsJson() throws JsonProcessingException {
```

Comment:
This is quite an interesting one. I'd be almost tempted to define it as a class variable, simply because the return is always the same and no arguments are passed in. For example: 

private static final String testContactsJson = objectMapper().writeValueAsString(
Arrays.asList(new Contact(2, "0207123456", "explanation 1", "explanation cy 1"),
            new Contact(5, "0207234567", "explanation 2", "explanation cy 2"),
            new Contact(9, "0207345678", "explanation 3", "explanation cy 3")
        );
);

Reply:
The ObjectMapper method is throwing a checked exception, hence doing this as a method :)